live-build/misc/ansible-roles/appliance-build.minimal-development/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+#
+# Copyright 2018 Delphix
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+- copy:
+    dest: /etc/cloud/cloud.cfg.d/99-delphix-development.cfg
+    mode: 0644
+    content: |
+      apt:
+        preserve_sources_list: false
```

Comment:
Just a side note: Looks like this is set to `false` by default.

Reply:
right.. so, in my platform change [here](https://github.com/delphix/delphix-platform/pull/11), we'll start setting that to true by default (i.e. we don't want use ubuntu's repos for customer installations or our QA variant), and then we'll override that setting for only the developer variant (via this change).